recipe/meta.yaml

```diff
@@ -1,92 +1,29 @@
-    - patches/cmake-find-ideep-mkldnn-in-sourcetree.patch # [x86] #required for mkl implementation
-    - patches/suppress_Wbitwise_instead_of_logical_clang.patch
+    # fixes linker error on linux platforms, see https://github.com/pytorch/pytorch/issues/90448
+    - patches/fix_kDefaultTimeout.patch
 
 build:
-  number: 1
+  number: 0
   skip: True  # [py<37]
-  # This is because this PR is only for GPU on osx-arm64. Will remove after merging
-  skip: True  # [not (osx and arm64)]
   # Dropping ppc because of various build issues
   skip: True  # [(linux and ppc64le)]
   string: gpu_cuda{{ cudatoolkit | replace('.', '') }}py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}  # [(pytorch_variant == "gpu") and (linux and x86_64)]
   string: gpu_mps_py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}                                      # [(pytorch_variant == "gpu") and (osx and arm64)]
   string: cpu_py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}                                          # [pytorch_variant == "cpu"]
   detect_binary_files_with_prefix: False
-  entry_points:
-    - convert-caffe2-to-onnx = caffe2.python.onnx.bin.conversion:caffe2_to_onnx
-    - convert-onnx-to-caffe2 = caffe2.python.onnx.bin.conversion:onnx_to_caffe2
-    - torchrun = torch.distributed.run:main
-  missing_dso_whitelist:
-    # It is not clear why conda-build cannot figure these out.
-    # It tries to find them in the site-packages for a different Python version from the one for which it's building.
-    # It would be nice to figure out why at some point.
-    - "**/ld64.so.*"              # [linux]
-    - "**/libc10.so"              # [linux]
-    - "**/libshm.so"              # [linux]
-    - "**/libtorch.so"            # [linux]
-    - "**/libtorch_cpu.so"        # [linux]
-    - "**/libtorch_python.so"     # [linux]
-    - "**/libiomp5.so"            # [linux]
-    - "**/libc10.dylib"           # [osx]
-    - "**/libshm.dylib"           # [osx]
-    - "**/libtorch.dylib"         # [osx]
-    - "**/libtorch_cpu.dylib"     # [osx]
-    - "**/libtorch_python.dylib"  # [osx]
-    - "**/libiomp5.dylib"         # [osx]
-    - "**/libc++.1.dylib"         # [osx]
-    - "**/asmjit.dll"             # [win]
-    - "**/c10.dll"                # [win]
-    - "**/fbgemm.dll"             # [win]
-    - "**/shm.dll"                # [win]
-    - "**/torch_cpu.dll"          # [win]
-    - "**/torch_python.dll"       # [win]
-    - "**/libcuda.so*"            # [(pytorch_variant == "gpu") and (linux and x86_64)]
-    - "**/libtorch_cuda.so"       # [(pytorch_variant == "gpu") and (linux and x86_64)]
-    - "**/libc10_cuda.so"         # [(pytorch_variant == "gpu") and (linux and x86_64)]
 
 requirements:
-  # WARNING:
-  #
-  # Make sure host and build Python are the same version. Unfortunately,
-  # PyTorch's build system (or some of the submodules) seems to mix up the
-  # Python interpreters.
-  #
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
```

Comment:
How about using `ninja-base` instead of `ninja` in `build`? Because the difference is an additional `python` dependency https://github.com/AnacondaRecipes/ninja-feedstock/blob/02e0641538afcc8657eeeea966b1c0c672f6231d/recipe/meta.yaml#L16-L36 but you already have `python` in `build`

Reply:
Sure, feels cleaner